express/server.js

```diff
@@ -60,7 +74,15 @@ Sentry.init({
     new Sentry.Integrations.Http({ tracing: true }),
     new Tracing.Integrations.Express({ app })
   ],
-  tracesSampleRate: 1.0
+  tracesSampleRate: 1.0,
+  tracesSampler: samplingContext => {
+    // sample out transactions for OPTIONS requests
+    if (samplingContext.request.method == 'OPTIONS') {
```

Comment:
When are `OPTIONS` requests being made within our app? Wasn't sure what situation led to this being added.

Reply:
These 'OPTIONS' http requests. I added the word 'http' if that makes it more clear.
Sometimes we open them during demo's by accident, and find that they're blank (there are no Spans inside of them). We have this in place in flask:
https://github.com/sentry-demos/application-monitoring/blob/master/flask/main.py#L40

This also allows us to have a valid example use case for tracesSampler ready in the app at all times, if we need to run an experiment for a customer.

https://sentry.io/organizations/testorg-az/performance/?project=6089805&statsPeriod=7d

The pre-flight 'OPTIONS' request to /products does not trigger the /products code to call the database (and hence no spans). It's not supposed to do that. It's just a pre-flight request, something related to CORS.
![image](https://user-images.githubusercontent.com/8920574/145116191-24c3863a-e702-4c93-a9e9-c990522ec0bd.png)

https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

@cstavitsky

Reply:
Thanks, was not familiar with the concept of Preflight requests, that answers my question